common/tests/CMakeLists.txt

```diff
@@ -43,32 +53,60 @@ if(WITH_COVERAGE)
   set(GCOV gcov)
 endif()
 
+
+file(COPY ${PROJECT_SOURCE_DIR}/tests/scripts/echo.bat
+     DESTINATION ${CMAKE_BINARY_DIR}/tests)
+file(COPY ${PROJECT_SOURCE_DIR}/tests/scripts/bad_script.bat
+     DESTINATION ${CMAKE_BINARY_DIR}/tests)
+
 add_test(NAME tests COMMAND ut_common)
 
 
+if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
+  target_link_libraries(
+    ut_common
+    PRIVATE centreon_common
+    centreon_http
+    centreon_process
+    -L${Boost_LIBRARY_DIR_RELEASE}
+    boost_program_options
```

Comment:
Wouldn't Boost::program_options be better?

Reply:
doesn't work on linux